h_transport_materials/property_database/niobium.py

```diff
@@ -67,13 +67,39 @@
     isotope="H",
 )
 
+sherman_diffusivity_h = Diffusivity(
+    D_0=4.4e-8 * u.m**2 * u.s**-1,
+    E_D=12.8 * u.kJ * u.mol**-1,
+    range=(
+        u.Quantity(860, u.degC),
+        u.Quantity(1120, u.degC),
+    ),
+    source="sherman_hydrogen_1983",
+    note="equation 13, range taken from figure 9",
+    isotope="H",
+)
+
+sherman_diffusivity_d = Diffusivity(
+    D_0=3.1e-8 * u.m**2 * u.s**-1,
+    E_D=14.0 * u.kJ * u.mol**-1,
```

Comment:
Regarding these values, this is given in the paper for diffusivity values T > 1100 K,

Reply:
yes but it doesn't give a range, hence I took the range from the figure